library/src/main/java/com/mikepenz/materialize/color/Material.java

```diff
@@ -48,5 +48,759 @@ public int getAsResource() {
+        _300("#A1887F", R.color.md_brown_300),
+        _400("#8D6E63", R.color.md_brown_400),
+        _500("#795548", R.color.md_brown_500),
+        _600("#6D4C41", R.color.md_brown_600),
+        _700("#5D4037", R.color.md_brown_700),
+        _800("#4E342E", R.color.md_brown_800),
+        _900("#3E2723", R.color.md_brown_900);
+
+        String color;
+        int resource;
+
+        Brown(String color, int resource) {
+            this.color = color;
+            this.resource = resource;
+        }
+
+        @Override
+        public String getAsString() {
+            return color;
+        }
+
+        @Override
+        public int getAsColor() {
+            return Color.parseColor(color);
+        }
+
+        @Override
+        public int getAsResource() {
+            return resource;
+        }
+    }
+
+    public enum Grey implements IColor {
+
+        _50("#FAFAFA", R.color.md_grey_50),
+        _100("#F5F5F5", R.color.md_grey_100),
+        _200("#EEEEEE", R.color.md_grey_200),
+        _300("#E0E0E0", R.color.md_grey_300),
+        _400("#BDBDBD", R.color.md_grey_400),
+        _500("#9E9E9E", R.color.md_grey_500),
+        _600("#757575", R.color.md_grey_600),
+        _700("#616161", R.color.md_grey_700),
+        _800("#424242", R.color.md_grey_800),
+        _900("#212121", R.color.md_grey_900);
+
+        String color;
+        int resource;
+
+        Grey(String color, int resource) {
+            this.color = color;
+            this.resource = resource;
+        }
+
+        @Override
+        public String getAsString() {
+            return color;
+        }
+
+        @Override
+        public int getAsColor() {
+            return Color.parseColor(color);
+        }
+
+        @Override
+        public int getAsResource() {
+            return resource;
+        }
+    }
+
+    public enum BlueGrey implements IColor {
+
+        _50("#ECEFF1", R.color.md_blue_grey_50),
+        _100("#CFD8DC", R.color.md_blue_grey_100),
+        _200("#B0BEC5", R.color.md_blue_grey_200),
+        _300("#90A4AE", R.color.md_blue_grey_300),
+        _400("#78909C", R.color.md_blue_grey_400),
+        _500("#607D8B", R.color.md_blue_grey_500),
+        _600("#546E7A", R.color.md_blue_grey_600),
+        _700("#455A64", R.color.md_blue_grey_700),
+        _800("#37474F", R.color.md_blue_grey_800),
+        _900("#263238", R.color.md_blue_grey_900);
+
+        String color;
+        int resource;
+
+        BlueGrey(String color, int resource) {
+            this.color = color;
+            this.resource = resource;
+        }
+
+        @Override
+        public String getAsString() {
+            return color;
+        }
+
+        @Override
+        public int getAsColor() {
+            return Color.parseColor(color);
+        }
+
+        @Override
+        public int getAsResource() {
+            return resource;
+        }
+    }
+
+    public enum Monochrome implements IColor {
```

Comment:
Not yet sure if Monochrome is the best thing. I would split up black and white as all other colors.